specutils/tests/test_loaders.py

```diff
@@ -1,25 +1,55 @@
-from astropy.utils.data import get_pkg_data_filename
+import pytest
 
-from .conftest import remote_data
 from .. import Spectrum1D
+from .conftest import remote_data_path
 
 
-def test_spectrum1d_GMOSfits():
-    optical_fits_file = get_pkg_data_filename('data/L5g_0355+11_Cruz09.fits')
-    optical_spec_2 = Spectrum1D.read(optical_fits_file, format='wcs1d-fits')
+@pytest.mark.parametrize('remote_data_path',
```

Comment:
Just another nit but you could probably use `functools.partial` to reduce code duplication somewhat.